content/docs/1.4.0/advanced-resources/data-recovery/recover-without-system.md

```diff
@@ -11,20 +11,47 @@ This command gives users the ability to restore a backup to a `raw` image or a `
 
 3. Specify the host path of output file by modifying field `hostpath` of volume `disk-directory`. By default the directory is `/tmp/restore/`.
 
-4. Set the first argument (backup url) by replacing `<BACKUP_URL>`, e.g. `s3://backupbucket@us-east-1/backupstore?backup=backup-bd326da2c4414b02&volume=volumeexamplename`. Do not delete `''`.
+4. Set the first argument (backup url) by replacing `<BACKUP_URL>`, e.g. `s3://<your-bucket-name>@<your-aws-region>/backupstore?backup=<backup-name>&volume=<volume-name>`. Do not delete `''`.
+
+    - `<backup-name>` and `<volume-name>` can be retrieved from backup.cfg stored in the backup destination folder, e.g. `backup_backup-72bcbdad913546cf.cfg`. The content will be like below: 
+
+        <pre><code>{"Name":"backup-72bcbdad913546cf","VolumeName":"volume_1","SnapshotName":"79758033-a670-4724-906f-41921f53c475"......}</code></pre>
```

Comment:
Hi @weizhe0422 :
    After I launched web, we couldn't see this contain on the web.
    
![Screenshot_20221101_181612](https://user-images.githubusercontent.com/104337648/199211575-fce2240b-4650-4a36-b992-69063f155a69.png)

Reply:
The 1.2.6, 1.3.2 and 1.3.3 are already updated, and the backup content from web page is also fixed to display.
![image](https://user-images.githubusercontent.com/17548901/199221083-e63e0be5-d519-4776-b202-62f22bf85568.png)